cmake/Config.cmake

```diff
@@ -39,21 +39,22 @@ elseif(CMAKE_SYSTEM_NAME MATCHES "^NetBSD$")
     set(SFML_OS_NETBSD 1)
     # don't use the OpenGL ES implementation on NetBSD
     set(OPENGL_ES 0)
+elseif(${CMAKE_SYSTEM_NAME} STREQUAL "iOS")
+    set(SFML_OS_IOS 1)
+    # As we want to find packages in our extlibs folder too
+    # we need to tell CMake to search there as well
+    set(CMAKE_FIND_ROOT_PATH_MODE_LIBRARY NEVER)
+    set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE NEVER)
+    set(CMAKE_FIND_ROOT_PATH_MODE_PACKAGE NEVER)
+    set(CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER)
```

Comment:
The comment says "our extlibs folder too", but then the option is set to `NEVER`, doesn't that mean, we're exclusively searching the `extlibs` folder?

Reply:
Yeah good spot - originally set to BOTH but then changed to NEVER, will update comment